facefusion/uis/components/webcam.py

```diff
@@ -97,19 +96,26 @@ def start(webcam_mode : WebcamMode, webcam_resolution : str, webcam_fps : Fps) -
 	source_image_paths = filter_image_paths(facefusion.globals.source_paths)
 	source_frames = read_static_images(source_image_paths)
 	source_face = get_average_face(source_frames)
+	ip_camera_url = ("rtsp://192.168.68.65:4545/h264.sdp")	#TESTED AND WORKING PEREFECTLY
```

Comment:
Android device with rtsp ip camera stream //worked on both local and remote networks. App is available also on iphone